### PR TITLE
More changes required in #73

### PR DESCRIPTION
**Pull request by joao7yt**
Added some spotted missing information as well as documenting the Properties and Style forms. Changed the position of a paragraph so the introduction to the Properties and Style are smoother.

If the formatting or language is not in the way you like it, you can tell me and I make the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,23 +92,101 @@ From edit mode, many operations are possible:
...
 
-Finally, when an element has a custom text position, it is annoying if the text position resets itself to some computed value whenever a boundary point or edge is moved. To prevent this, there is the *update text position* menu, which when unchecked, leaves the text position as it is whenever a manipulation is done that is not moving the entire element.
+When right clicking an element, or a boundary point or edge of an element, the context menu will contain options for editing the element. These options include opening the properties and style window, removing the entire element, adding boundary points, if an edge is highlighted, and removing a boundary point if one is highlighted. The *move* menu allows you to move elements on top of others, i.e. change the z-index of the element. The properties and style for the entire keyboard can always be opened from the right click menu while in edit mode.
 
 All actions can be undone using `Ctrl` + `Z`, and redone using `Ctrl` + `Shift` + `Z`.
 
+#### Properties and Style
```
Comment (ThoNohT) [inline README.md]: Perhaps include "windows" in the title to signify you're describing the windows.

File: README.md [hunk 2]
```diff
@@ -92,23 +92,101 @@ From edit mode, many operations are possible:
...
+    - **General**  
+The *Color 1* refers to the color of the mouse speed indicator outer circle as well as the color of the direction indicator when the mouse is moving slowly. The *Color 2* refers to the color of the direction indicator when the mouse is moving quickly. The mouse speed is relative to the *Mouse sensitivity* value in the *Settings*. You can also change its width.
+
+- **Elements Properties**
+  - **Keyboard Key Properties**
+    - **Text**/**Shift Text**  
+Text shown on the element when, respectively, the 'Shift' key isn't/is pressed or 'Caps Lock' isn't/is activated.
+
+    - **Text Position**  
+Text position relative to the keyboard itself. The *Center* button automatically calculates the center of the element.
```
Comment (ThoNohT) [inline README.md]: To the key itself.

File: README.md [hunk 3]
```diff
@@ -92,23 +92,101 @@ From edit mode, many operations are possible:
...
+
+    - **Key codes**  
+Here you are presented with a TextBox where you can enter a key-code. You can add this new key-code by using the *Add* button. The boundary list in right below where you can select each key-code and remove it. The *Detect* button is there to make your life easier allowing you to detect a specific key-code of a key by just pressing the key on your physical keyboard. By adding more than one key-code you can make special shortcut elements, e.g., by adding the 162 and 65 key-codes, you can make an element that will light up when 'Ctrl' + 'A' is used.
+
+  - **Mouse Key Properties**
+    - **KeyCode**  
+In the drop-down list you can choose which mouse button will light up the element. Currently there're 5 options: *LeftButton*, *MiddleButton*, *RightButton*, *X1Button* and *X2Button* (these last two being the side buttons of the mouse).
```
Comment (ThoNohT) [inline README.md]: there're.. is that really a word? I'd say there are here in writing

**Comment by ThoNohT** [comment]
I added some comments with suggestions to improve the text. Overall it looks good. I'm also going to look into enabling a wiki on here, so everyone can just contribute their changes to the documentation.

File: README.md (hunk 2 above)
Comment (joao7yt) [reply README.md]: Isn’t this position relative to the 0 ; 0 position of the keyboard?

**Comment by joao7yt** [comment]
I reviewed two of the reviews because the text position one I think is right. The text position is relative to the 0 ; 0 position of the keyboard and not the element.

**Comment by ThoNohT** [comment]
I think you are right yeah.

**Pull request merged**